Podcast/EmptyStateTableView.swift

```diff
@@ -71,6 +77,24 @@ class EmptyStateTableView: UITableView, EmptyStateViewDelegate {
         emptyStateTableViewDelegate?.didPressEmptyStateViewActionItem()
     }
     
+    //MARK
+    //MARK: Refresh Control Functions
+    //MARK
+    
+    func endRefreshing() {
+        if let control = refreshControl {
+            if control.isRefreshing {
+                DispatchTime.waitFor(milliseconds: 500, completion: { self.refreshControl?.endRefreshing() } )
```

Comment:
why is there a delay of 500 ms?

Reply:
We need this for the refresher not too be on top of the table view during refresh. It's what solves the bug. The problem is, when we refresh views that result in no changes it returns too fast.